tests/flytekit/integration/remote/test_rust_remote.py

```diff
@@ -0,0 +1,81 @@
+import importlib
+import sys
+import time
+
+import pytest
+
+from flytekit import task
+from flytekit.configuration import Config, ImageConfig, SerializationSettings
+from flytekit.remote import FlyteRemote
+from flytekit.remote.entities import FlyteTask
+
+PROJECT = "flytesnacks"
+DOMAIN = "development"
+
+TASK_NAME = "tests.flytekit.integration.remote.test_rust_remote.my_test_task"
+VERSION_ID = f"{hash(time.time())}"  # we use current timestamp when initialize tests, to prevent identical re-register.
+
+
+def check_python_installation(module_name):
+    try:
+        importlib.import_module(f"{module_name}")
+        if f"{module_name}" in sys.modules:
```

Comment:
From what I Know, `sys.modules` returns a dictionary that contains all the modules that have been imported since the program started.

I found some tests check whether a module is installed or not by doing [so](https://github.com/flyteorg/flytekit/blob/cc3a7a9277852dc06bc1b2c041a962be973d8faf/tests/flytekit/unit/core/test_dataclass.py#L15C22-L15C28).

```python
@pytest.mark.skipif("pandas" not in sys.modules, reason="Pandas is not installed.")
```

I'm curious is this intended?